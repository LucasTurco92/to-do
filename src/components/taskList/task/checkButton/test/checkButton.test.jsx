import { render } from '@testing-library/react';
import CheckButton from '../checkButton';

it('When it invokes the CheckButton component, should render it properly', ()=>{

    const { container } = render(<CheckButton />);
    
    expect(container).toMatchSnapshot();
});
