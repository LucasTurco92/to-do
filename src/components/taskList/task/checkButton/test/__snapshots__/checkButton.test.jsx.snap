// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When it invokes the CheckButton component, should render it properly 1`] = `
<div>
  <div
    class="sc-bdvvtL ivoDIz"
  />
</div>
`;
